index.html

```diff
@@ -531,9 +531,9 @@ <h3>Accessible Name Calculation</h3>
 				<dd>One of the following values:
 					<ol>
 						<li>author: name comes from values provided by the author in explicit markup features such as the <pref>aria-label</pref> attribute, the <pref>aria-labelledby</pref> attribute, or the host language labeling mechanism, such as the <code>alt</code> or <code>title</code> attributes in <abbr title="Hypertext Markup Language">HTML</abbr>, with HTML <code>title</code> attribute having the lowest precedence for specifying a text alternative.</li>
+            <li>caption/legend: name comes from the text value of the first descendant <a>element</a> node with the role of <code>legend</code> or <code>caption</code>.  In the case of the role <code>caption</code> name is also allowed from the last child node of the <a>element</a>.  Although "caption" or "legend" may be allowed in addition to "author" in some <a>roles</a>, "caption" or "legend" is used in content only if higher priority "author" features are not provided. Priority is defined by the <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te">accessible name and description computation</a> algorithm.  [[ACCNAME-1.1]]</li>
```

Comment:
is it necessary to state here where the caption/legend is placed beyond it must be a direct child of the parent element?  I would think we can just indicate the appropriate placement for the elements specifically in their definition / their parent element's definition.

e.g. mention in `group`, `radiogroup`, `grid` and `table` that a `caption/legend` must be the first child element.  mention in `figure` that `caption/legend` must be the first or last child element.

Reply:
@scottaohara I think it gives authors a better understanding of the usage of the caption/legend technique and is pretty succinct, so I would advocate leaving it.  It will help people understand how it is different form other labeling techniques that are in the list of naming techniques, without having to read the details for specific roles that can be labeling with caption/legend.

Reply:
understand the desire to leave it in, but then there should be better clarity that this statement only applies to `figure`:
>In the case of the role `caption` name is also allowed from the last child node of the element

Reply:
@scottaohara changed to say "in some cases" for the last child role